src/main/java/com/prgrms/commandLineApplication/controller/VoucherController.java

```diff
@@ -37,33 +40,51 @@ public void run() {
 
         switch (menuType) {
           case EXIT -> isRunning = false;
-          case LIST -> list();
-          case CREATE -> create();
+          case VOUCHERLIST -> voucherList();
+          case CREATEVOUCHER -> createVoucher();
+          case CUSTOMERLIST -> customerList();
+          case CREATECUSTOMER -> createCustomer();
         }
       } catch (IllegalArgumentException e) {
-        LOGGER.error("Error Message => {}", e.getMessage());
+        logger.error("Error Message => {}", e.getMessage(), e);
         console.printErrorMessage(e);
       } catch (NoSuchElementException e) {
-        LOGGER.warn("Warn Message => {}", e.getMessage());
+        logger.warn("Warn Message => {}", e.getMessage(), e);
         console.printErrorMessage(e);
       }
     }
   }
 
-  private void list() {
+  private void voucherList() {
     List<Voucher> list = voucherService.findAllVouchers();
-    console.printAllVoucher(list);
+    console.printAllVouchers(list);
   }
 
-  private void create() {
+  private void createVoucher() {
     console.requestVoucherType();
     String voucherType = console.readVoucherType();
 
     console.requestDiscountAmount();
     int discountAmount = console.readVoucherAmount();
 
     voucherService.create(voucherType, discountAmount);
-    console.printCreateSuccess(voucherType, discountAmount);
+    console.printCreateVoucherSuccess(voucherType, discountAmount);
+  }
+
+  private void customerList() {
+    List<Customer> list = customerService.findAllCustomers();
+    console.printAllCustomers(list);
+  }
```

Comment:
```suggestion
  private void printCustomers() {
    List<Customer> list = customerService.findAllCustomers();
    console.printAllCustomers(list);
  }
```
메서드는 동사나 동사구로 작성하는 것이 좋아보이네요

Reply:
936f27d